workers/unity/Packages/com.improbable.gdk.core/Worker/ViewDiff.cs

```diff
@@ -238,12 +243,39 @@ public void AddEntityQueryResponse(WorldCommands.EntityQuery.ReceivedResponse re
             worldCommandsReceivedStorage.AddResponse(response);
         }
 
+        public void AddLogMessage(string message, LogLevel level)
+        {
+            logsReceived.Add(new LogMessageReceived(message, level));
```

Comment:
This does mean we can't use the same diff object for sending & receiving - although that's probably okay

Reply:
We already can't because of commands for now :(.